### DISCUSSION TITLE
Wrong translation (portuguese)

### DISCUSSION THREAD

**Discussion by mateus10011001** [WinMerge/winmerge#572] [General]
Hi! The translation of "Copy Right" at the tooltip of the "folder compare toolbar" is currently "Copiar da direita"(copy from right), but should be "Copiar para direita"(copy to right) ! Seems a minor error but of course completely changes what it means in a way that if someone(almost happened with me rs) that only pays attention to the tip may erease data unintentionally. Same to "Copy Left"/"Copar para esquerda" 

![image](https://user-images.githubusercontent.com/60583155/105606732-e0455c00-5d79-11eb-95ca-54b4bef81fa8.png)

Great app! Congrats!

**Comment by sdottaka** [comment]
Fixed by commit 082abc7bc07078c96434c3cf073389b4c13d8832

Thank you for the report.

**Comment by mateus10011001** [comment]
:)

Em sáb, 23 de jan de 2021 20:07, Takashi Sawanaka <notifications@github.com>
escreveu:
